crates/ln-dlc-node/src/config.rs

```diff
@@ -16,7 +16,7 @@ pub const LIQUIDITY_MULTIPLIER: u64 = 2;
 /// Sats/vbyte rate for position
 ///
 /// The coordinator and the app have to align on this to agree on the fees.
-pub const CONTRACT_TX_FEE_RATE: u64 = 4;
+pub const CONTRACT_TX_FEE_RATE: u64 = 7;
```

Comment:
🤔 that's very conservative.. I would go for something like 20, to be on the safe side - since this value can't be changed for a stuck dlc channel afterwards.

Reply:
7 seems to look ok-ish for now, but it's not ideal as we do not know how the mempool situation develops. 
20 seems to be an overkill for me as we would always overpay.